xl-v1.go

```diff
@@ -175,6 +162,8 @@ func newXLObjects(disks []string) (ObjectLayer, error) {
 		dataBlocks:    dataBlocks,
 		parityBlocks:  parityBlocks,
 		listPool:      newTreeWalkPool(globalLookupTimeout),
+		// Initialize object caching, FIXME: cache has no expiration for the time being.
+		objCache: objcache.New(maxCacheSize, objcache.NoExpiration),
 	}
```

Comment:
globalCacheSize should be initialized to maxCacheSize or MINIO_CACHE_SIZE environment variable.